SU-SWS/stanford-jumpstart-deployer
# Stable20170525

# READY FOR REVIEW

# Summary
- These commits bring module versions up to the stable20170525 gitolite branch.

# Needed By (Date)
- I don't believe there is a required by date.

# Urgency
- It'd be good to merge this sooner rather than later, just so we can be sure future changes to the deployer are consistent with this state.

# Steps to Test
- See test results below.  Please let me know if you would like a fuller suite of tests run on sites-uat.  Travis used the deploy tag, so that tests can complete within the roughly 50m time period given.

# Associated Issues and/or People
- https://stanfordits.atlassian.net/projects/DEVOPS/issues/DEVOPS-16

We'll also want to review and merge these pull requests for DEVOPS-16.
- https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/145
- https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/144
